string limb
float64[] joint_velocity
---
int64 success_bool
